identity/app/views/fragments/profile/accountDetailsForm.scala.html

```diff
@@ -3,7 +3,7 @@
 @import views.html.fragments.form.{inputField, textareaField}
 @import form.IdFormHelpers._
 @import form.Input
-@import model.Countries
+@import model.{Countries, CountryCodes}
 @import org.joda.time.DateTime.now
 @import helper._
 
```

Comment:
Currently the 2 submit buttons in one form mean that it is not possible to use the enter button to submit updated details as the browser will just use the first submit button in the form and delete the phone number instead.

It is possible to get round this by adding a hidden button at the top of the form:
<button type="submit" class="is-hidden"></button>